gsa_admin.py

```diff
@@ -457,6 +457,56 @@ def syncDatabases(self, database_list):
       except:
         log.error("Unable to sync %s properly" % database)
 
+  def pauseCrawl(self):
+    """Pause crawl on the GSA.
+       Supports only 7.2 and higher.
+
+    Args:
+      None
+    """
+    self._login()
+    security_token = self.getSecurityToken('exportAllUrls')
```

Comment:
Why are you getting the securityToken from 'exportAllUrls' and not from 'crawlStatus'